README.md

```diff
@@ -82,6 +88,25 @@ class User < ActiveRecord::Base
     # Optional flag to keep empty parent directories when deleting files.
     :ftp_keep_empty_directories => true # optional, false by default
   }
+
+  # If you are using 2nd method to add FTP Servers.
+  def ftp_credentials
+    [
+      {
+        :host     => "ftp1.example.com",
+        :user     => "foo",
+        :password => "bar"
+      },
+      # Add more servers if needed
+      {
+        :host     => "ftp2.example.com",
+        :user     => "foo",
+        :password => "bar",
+        :port     => 2121, # optional, 21 by default
+        :passive  => true  # optional, false by default
+      }
+    ]
```

Comment:
Instead of duplicating the static array from above I suggest to comment out this method and provide an artificial dynamic example:
```ruby
# def ftp_credentials
#   # return ftp server(s) to use for the current attachment
#   [
#     {
#       :host     => "ftp#{id % 3}.example.com",
#       :user     => "foo",
#       :password => "bar"
#     }
#   ]
# end
```